src/atoms.ts

```diff
@@ -255,45 +279,45 @@ function parseAtom(
                 id: audioId,
                 trackUrl,
                 kicker,
-                title
+                title,
             });
         }
 
         case "quiz": {
-            const atom = atoms.quizzes?.find(quiz => quiz.id === id);
-            if (atom?.data?.kind !== "quiz" || !id) {
+            const atom = atoms.quizzes?.find((quiz) => quiz.id === id);
+            if (atom?.data.kind !== "quiz" || !id) {
                 return err(`No atom matched this id: ${id}`);
             }
 
-            const { content } = atom?.data?.quiz;
+            const { content } = atom.data.quiz;
 
-            if (!content) {
+            if (content.questions.length === 0) {
```

Comment:
@webb04 did you make this change or did it do it automatically?

Reply:
I made this one, it required it to be a more specific check